client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -621,7 +621,7 @@ export class DomainWarnings extends PureComponent {
 			( { registrationDate } ) =>
 				registrationDate && moment( registrationDate ).add( 2, 'days' ).isAfter()
 		);
-		if ( this.props.isSiteEligibleForFSE && this.props.siteIsUnlaunched && isWithinTwoDays ) {
+		if ( this.props.siteIsUnlaunched && isWithinTwoDays ) {
```

Comment:
It seems previously this condition was only met for FSE eligible sites that were unlaunched, now with removing that first check this condition is met for all sites that are unlaunched?

Im not entirely sure what makes the most sense here: removing this entire conditional block (probably?), leaving it in with some legacy FSE active check, or other?

Reply:
Thanks for checking! I will investigate, but my hunch is that complete removal is the way.

Reply:
Chased this down to #37960 and you're right, I get to delete moar things ␡